src/include/units/quantity.h

```diff
@@ -146,6 +146,9 @@ class quantity {
   quantity& operator=(const quantity&) = default;
   quantity& operator=(quantity&&) = default;
 
+  template <PointOrigin Orig>
+  [[nodiscard]] constexpr quantity_point<dimension, unit, rep, Orig> absolute() const noexcept;
```

Comment:
I am not sure if we want quantity to be aware of `quantity_point`?

Reply:
Up to bike-shedding too.

Pros:
 - Symmetry with `quantity_point::relative`
 - It is otherwise non-trivial to instantiate `quantity_point` from a `quantity` with matching units and a specific origin: instead of `radar_distance_reading.template absolute<ground_level>()` in generic code, I would have to write
 ```
units::quantity_point<typename decltype(radar_distance_reading)::dimension, typename decltype(radar_distance_reading)::unit, typename decltype(radar_distance_reading)::rep, ground_level>(radar_distance_reading)
 ```

Con:
 - There is some circular dependency here, with the implementation of that method deferred to `quantity_point.h`.

Alternatives/questions:
 - We could instead provide a free-standing function to that end
 - We could make the point_origins to be constants so that the `template` keyword can be dropped and provided as method argument instead
 - We could provide suitable constructors with the right deduction guides

Reply:
`quantity_point{1 * m}` already works and uses the default origin.
How about
```C++
template<PointOrigin> QuantityPoint auto absolute(Quantity auto);
QuantityPoint auto absolute(Quantity auto); // default origin
template<PointOrigin> QuantityPointKind auto absolute(QuantityKind auto);
QuantityPointKind auto absolute(QuantityKind auto); // default origin
```

Reply:
Free-standing `absolute` function is fine with me. I'm wary of default origins though:

Default origins only works when there is a single unambiguous origin. In those cases, half the reason to use quantity points at all is gone. (the other half is to make it clear that one is talking about points, not distances - which is still useful).

But one of our prime use-cases is one where there are different origins (temperatures): Currently, you would pass the following: `assert(quantity_point(1 * deg_C) == quantity_point(1 * K))`, which is most likely not what you want. You could argue that this is in fact correct. Alternatively the constructor could be adjusted (with some effort) so that instead the following would pass: `assert(quantity_point(0 * deg_C) == quantity_point(273.15 * K))`. That is potentially less surprising. But exactly that ambiguity which of those two asserts even should be correct is the reason why I really would like to avoid default origins at all. In generic code, that becomes a nightmare, where the result of a piece of code becomes physically different dependent on which of those physically equivalent units are used.

Reply:
I suppose it'd be fine to leave them out until someone needs them.

Reply:
> In those cases, half the reason to use quantity points at all is gone.

I do not agree. Please see the `glide_computer` example.

If we prefer to have just 2 overloads we could provide a default origin for `PointOrigin` but it will result in many diamond operators `<>` in the code.

1. As a result, should we disable CTAD for `quantity_point` from a `quantity`?
2. Should we provide the non-member `relative()` function as well (while still keeping the member `relative()` one)?

Reply:
`assert(quantity_point(1 * deg_C) == quantity_point(1 * K))` is obviously bad and should fail.

Reply:
So then I am in favour of disabling CTAD for quantity_point from quantity.

Reply:
OK, let's do it.